ddd-kata.md

```diff
@@ -0,0 +1,94 @@
+# Domain-Driven Design Kata
+
+## What is a Kata?
+
+A kata is a specific sequence of movements and techniques practiced in the martial arts. The word "kata" is Japanese and is used in many martial arts originating from Japan, such as karate, judo, and aikido. Kata is usually taught by a sensei, who will demonstrate the movements and then guide the students as they practice the kata. The practice of kata can help martial artists to develop proper technique, balance, and control, and to learn the application of various techniques in a practical context. Some martial arts styles have kata that are performed solo, while others require a partner.
+Additionally, Kata is also commonly used in software development, to practice writing software that accomplish specific tasks, by following a predetermined set of steps. The goal of the practice is to create software that is reliable, easy to maintain and flexible. These principles are now applied in practicing Domain-Driven Design.
+
+## Follow the DDD starter modelling process
+
+![DDD starter Modelling Process (CC-BY-SA-4.0 license)](./images/ddd-starter-modelling-process-colored.png)
+
+This [process](https://github.com/ddd-crew/ddd-starter-modelling-process) walks you through each stage of understanding and executing Domain-Driven Design (DDD), from orienting around an organization's business model to coding a domain model.
+
+This approach will walk you through each of the critical processes in developing a software system with the DDD perspective, allowing you to focus on your business concerns rather than being overwhelmed by understanding DDD.
```

Comment:
This sentence is kinda redundant to the previous one.

Reply:
Removed